source/src/showSpeed.kt

```diff
@@ -1,7 +1,11 @@
-class Speed (val _speed: Int){
-    val speed = _speed
-
+data class Speed (val speed: Int ){
+    constructor(): this(0)
```

Comment:
U can make default value in constructor 
Try this
`data class Speed (val speed: Int = 0){`